MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -334,15 +450,15 @@ void AddSdksFromDir (string sdkDir)
 			var items = new List<CompletionItem> ();
 
 			if (comparandVariables != null && isValue) {
-				foreach (var ci in GetComparandCompletions (doc, comparandVariables)) {
-					items.Add (CreateCompletionItem (ci));
+				foreach (var ci in ExpressionCompletion.GetComparandCompletions (doc, comparandVariables)) {
+					items.Add (CreateCompletionItem (ci, MSBuildSpecialCommitKind.AttributeValue));
 				}
 			}
 
-			if (isValue) {
+		if (isValue) {
 				switch (kind) {
 				case MSBuildValueKind.NuGetID:
-					return await GetPackageNameCompletions (doc, triggerLocation.Position - triggerLength, triggerLength);
+					return await GetPackageNamesAsync (doc, rr);//GetPackageNameCompletions (doc, rr, triggerLocation.Position - triggerLength, triggerLength);
```

Comment:
I don't think the method needs to be renamed, and even if it did, there's no need for the trailing comment :)

Reply:
I removed the trailing comments and changed methods in my most recent commit.